scripts/testdata/expected-directpath.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "gke-pod-network",
+  "cniVersion": "0.3.1",
+  "plugins": [
+    {
+      "type": "ptp",
+      "mtu": 1460,
+      "ipam": {
+        "type": "host-local",
+        "ranges": [
+          [{"subnet": "10.52.1.0/24"}], [{"subnet": "2600:1900:4000:318:0:7:0:0/112"}]
+        ],
+        "routes": [
+          {"dst": "0.0.0.0/0"}, {"dst": "::/0"}
```

Comment:
This is not the expected directpath (meaning you didn't get the env right).

Directpath doesn't have a default IPv6 route. (only one to directpath prefix)

Reply:
so it should use env `CNI_SPEC_IPV6_ROUTE` - right?

Reply:
right.

Reply:
fixed